Dockerfile

```diff
@@ -1,34 +1,40 @@
-FROM ubuntu
-RUN apt-get update && apt-get -y install curl sudo procps wget && \
-    echo "%sudo ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
-    useradd -u 1000 -G users,sudo -d /home/user --shell /bin/bash -m user && \
-    echo "secret\nsecret" | passwd user && \
-    curl -sSL https://get.docker.com/ | sh && \
-    # Add user to docker group: 100 and 50 are this gIDs in boot2docker and Docker for Mac
-    usermod -aG docker,100,50 user && \
-    sudo apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+FROM alpine:3.4
```

Comment:
Do we need to add some comments before the FROM to explain how to run this Dockerfile ?